modules/openshift-developer-cli-using-command-completion.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * cli-reference/odo-cli-reference.adoc    
+
+[id="using-command-completion_{context}"]
+= Using command completion
+
+[NOTE]
+====
+* Currently command completion is only supported for bash, zsh, and fish shells.
```

Comment:
I don't think it's good practice to put bullets inside a note. I'd either combine into two successive sentences, or if they are too unrelated, make two separate notes. 

I know it's not great looking to have two separate notes back-to-back, but maybe the second note could be moved to after the procedure, since it does apply to after you've enabled it.